src/pymodaq_plugins_keithley/daq_viewer_plugins/plugins_0D/daq_0Dviewer_Keithley2100.py

```diff
@@ -152,22 +112,20 @@ def grab_data(self, Naverage=1, **kwargs):
         =============== ======== ===============================================
 
         """
+        logger.info("grab_data called")
         data = self.controller.read()
         dte = DataToExport(
             name="K2100",
             data=[
-                DataFromPlugins(
-                    name="K2100", data=data, dim="Data0D", labels=["dat0", "data1"]
-                )
+                DataFromPlugins(name="K2100", data=data, dim="Data0D", labels=["dat0", "data1"])
```

Comment:
the data attribute should be a list of numpy array. is that the case from the data = self.controller.read() line?

Reply:
Is there two channels? The number of elements in labels should reflect this with appropriate names!